core/src/main/java/com/datastrato/gravitino/storage/relational/po/CatalogPO.java

```diff
@@ -104,82 +104,91 @@ public int hashCode() {
   }
 
   public static class Builder {
-    private final CatalogPO metalakePO;
+    private final CatalogPO catalogPO;
 
-    public Builder() {
-      metalakePO = new CatalogPO();
+    private Builder() {
+      catalogPO = new CatalogPO();
     }
 
     public CatalogPO.Builder withCatalogId(Long catalogId) {
-      metalakePO.catalogId = catalogId;
+      catalogPO.catalogId = catalogId;
       return this;
     }
 
     public CatalogPO.Builder withCatalogName(String name) {
-      metalakePO.catalogName = name;
+      catalogPO.catalogName = name;
```

Comment:
Could you please help to review the changes in this class? thanks! @yuqi1129

Reply:
@mchades It's okay with me for the changes.